src/redux/reducers/reducer.js

```diff
@@ -0,0 +1,19 @@
+
```

Comment:
You have to take out the redux folder here too. I also don't expect all reducer to be on one file, so if the action is called homeAction.js, I expect you name this homeReducer.js, then create index.js where all the reducers created can be imported and combine using combineReducers from redux.

Reply:
@Ruqoyah you mean to take the redux folder outside of the src folder? besides am done with creating and index.js with combineReducers.